tensorflow/lite/micro/tflite_bridge/BUILD

```diff
@@ -33,7 +35,7 @@ cc_library(
     hdrs = ["op_resolver_bridge.h"],
     copts = micro_copts(),
     visibility = [
-        "//visibility:public",
+        "//tensorflow/lite/micro:__pkg__",
```

Comment:
want to change this to "//tensorflow/lite/micro:__subpackages__"? or do you want to explicitly prevent those from using flatbuffer_conversions_bridge?

https://bazel.build/concepts/visibility#visibility-specifications

Reply:
I think preventing is a good idea. The reason is, micro/subpackages are very low-level code like arena and memory_planning. These high-level API are used mostly in MicroGraph, MicroAllocator, etc to populate/process node_registrations, etc. which are slightly higher in our design.

So, the current state looks good to me. We can change if we ever refactor the code, thoughts?

Reply:
sgtm